From 9b73248b9c6db050dacf38776e2d1554725a4a96 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Tue, 12 Mar 2024 12:38:50 +0800
Subject: [PATCH 07/10] FROM AOSC: fix(dxvk-native-1.9.2a): fix build with GCC
 >= 13
X-Developer-Signature: v=1; a=openpgp-sha256; l=477; i=xtexchooser@duck.com;
 h=from:subject; bh=IkqvfKlJEUz173wTwClN/YTu7XnQt0mai9FLNdJ+b10=;
 b=owGbwMvMwCW2U4Ij7wZL9ETG02pJDGkvVLb+1vr6PPf3hdNcUuubnPWPeTcq7ZnLGHuzkk1nd
 8tk3dC/HaUsDGJcDLJiiixFhg3erDrp/KLLymVh5rAygQxh4OIUgImcn8Dw33063/SO1yc2hkmf
 9ksz6kz5vvrokceMugZN57M0rqzLFGFkmN6Ytfev1iTtBS3i+/RLn/Ofm+V84Il96NeDRieeW+e
 eYwQA
X-Developer-Key: i=xtexchooser@duck.com; a=openpgp;
 fpr=7231804B052C670F15A6771DB918086ED8045B91

---
 src/libs/dxvk-2.3.1/src/util/util_vector.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/libs/dxvk-2.3.1/src/util/util_vector.h b/src/libs/dxvk-2.3.1/src/util/util_vector.h
index 08fea02f2827..2f431be5c322 100644
--- a/src/libs/dxvk-2.3.1/src/util/util_vector.h
+++ b/src/libs/dxvk-2.3.1/src/util/util_vector.h
@@ -1,6 +1,7 @@
 #pragma once
 
 #include <iostream>
+#include <cstdint>
 
 #include "util_bit.h"
 #include "util_math.h"
-- 
2.46.0

